src/assets/JSONLDAsset.js

```diff
@@ -0,0 +1,35 @@
+const Asset = require('../Asset');
+
+// A list of all attributes in a schema that may produce a dependency
+// Based on https://schema.org/Place
+const SCHEMA_ATTRS = [
+  "logo",
+  "photo",
+  "image"
+];
```

Comment:
Good start. I guess we should go through the specs and find all the possible attributes that are URLs

Reply:
Not sure if they list every single property, but https://schema.org/ImageObject has a dependents list: "Instances of ImageObject may appear as values for the following properties"